data/admin.py

```diff
@@ -84,11 +84,16 @@ def get_queryset(self, request):
 
     def cohorte(self, obj):
         coh = obj.suggestion_cohorte
-        return format_html(f"{coh.identifiant_action}<br/>{coh.identifiant_execution}")
+        return format_html(
+            "{}<br/>{}", coh.identifiant_action, coh.identifiant_execution
+        )
 
     def acteur_link_html(self, id):
-        return f"""<a target='_blank'
-        href='/admin/qfdmo/displayedacteur/{id}/change/'>{id}</a>"""
+        return format_html(
+            '<a target="_blank" href="/admin/qfdmo/displayedacteur/{}/change/">{}</a>',
```

Comment:
je suis pas hyper chaud de construire l'url à la main, pourquoi est-ce qu'on ne récupère pas `change_url` plutôt ?

Reply:
si on ne veut pas instancier l'acteur, sinon on peut créer une méthode static 

```py
class DisplayedActeur:
  @staticmethod
  def get_change_url(id):
    return reverse(
             "admin:qfdmo_displayedacteur_change", args=[quote(id)]
         )
  @property
  def change_url(self): 
    return self.get_change_url(self.identifiant_unique)
```

comme ça ici on peut appeler 
```suggestion
            '<a target="_blank" href="{}">{}</a>', DisplayedActeur.get_change_url(id), id 
```

Reply:
et on évite de reconstruire manuellement l'url (qui est vraiment fragile)

Reply:
done, j'ai créé la property uniquement